doc/classes/ArrayMesh.xml

```diff
@@ -185,6 +186,7 @@
 	</methods>
 	<members>
 		<member name="blend_shape_mode" type="int" setter="set_blend_shape_mode" getter="get_blend_shape_mode" enum="Mesh.BlendShapeMode" default="1">
+			Sets the blend shape mode to one of [enum Mesh.BlendShapeMode]. See [enum Mesh.BlendShapeMode] for options.
```

Comment:
Maybe a bit redundant to mention the enum twice? Doesn't hurt though, and I guess you did it to be consistent with the "See enum for options." format.

Reply:
It is a bit awkward. I'll remove it.